onnxruntime/core/providers/dml/DmlExecutionProvider/src/External/DirectMLHelpers/StringifyHelpers.h

```diff
@@ -0,0 +1,54 @@
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+#pragma once
+
+#pragma warning(push)
+#pragma warning(disable:4499)
+
+// This should only include ToString functions for enums which are not part of DirectMLSchema.json
+namespace StringifyHelpers
+{
+    template <typename T>
+    static const char* ToString(T value)
+    {
+#ifndef WAI_BUILD_LINUX
```

Comment:
>WAI_BUILD_LINUX [](http://example.com/codeflow?start=8&length=15)

Although it doesn't hurt to leave this hurt, it doesn't apply either, right? I don't believe ORT ever sets this on Linux (or that ORT builds with the DML EP currently ... even though it could because DML runs on WSL).

Reply:
Yes, you are right. It doesn't apply here.